mathematica/cohen_hubbard_oesterwinter.wl

```diff
@@ -39,17 +36,17 @@ With[
   {j, 0, n - 2}]]];
 
 
-{{LCM@@(Denominator/@#),#*LCM@@(Denominator/@#)}}&[\[Beta]]//TableForm
+{{LCM@@(Denominator/@#),#*LCM@@(Denominator/@#)}}&[\[Beta][14]]//TableForm
 
 
-(fs[-h,\[ScriptCapitalN]+3]-fs[-2h,\[ScriptCapitalN]+3])/h+h Sum[f2s[-i h,\[ScriptCapitalN]+3]\[Beta][[i]],{i,1,Length[\[Beta]]}]//Expand
+ReleaseHold[Hold[(fs[-h,\[ScriptCapitalN]+3]-fs[-2h,\[ScriptCapitalN]+3])/h+h Sum[f2s[-i h,\[ScriptCapitalN]+3]\[Beta][\[ScriptCapitalN]][[i]],{i,1,Length[\[Beta][\[ScriptCapitalN]]]}]]/.\[ScriptCapitalN]->14]//Expand
```

Comment:
What is going on with this ReleaseHold and Hold?

Reply:
_Mathematica_ magic: (1) Hold prevents the functions like `fs` from being evaluated at a place (2) `N` is being substituted in the argument of `fs` et al. (3) releasing the hold causes the whole thing to evaluate.